android-components/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
+# this is the network timeout default value in ms.
+networkTimeout=10000
```

Comment:
❓ Do we need to set it if it's already set by default to 10s?

Reply:
Not necessarily, but this is a change in the new version, the ability to configure network timeout. I added this here for surfacing the new configuration param and its default value.